### PR TITLE
(DOCSP-644): Removed Windows as a Service install.

### PR DESCRIPTION
**Pull request by atsansone**
@i80and : Please review and approve. It just removes the Windows install as a service.

[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/anthonysansone/DOCSP-644/installation.html)

### CODE REVIEW

File: source/includes/steps-install-bi-connector.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ action:
          .. cssclass:: copyable-code
          .. code-block:: powershell
 
-            move <donwloadDirectory>/bin/* "C:\Program Files\MongoDB BI Connector"
+            move {downloadDirectory}\* "C:\Program Files\MongoDB BI Connector"
```
Comment (i80and) [inline source/includes/steps-install-bi-connector.yaml]: Is there a reason you used curly braces here? I thought our house style was angle brackets

Comment (i80and) [inline source/includes/steps-install-bi-connector.yaml]: Also, this isn't in the system ``PATH``, which makes the line above a bit of a lie

Comment (i80and) [inline source/includes/steps-install-bi-connector.yaml]: And last but not least, don't we have to create that directory?

File: source/includes/steps-install-bi-connector_fixed.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+title: Uninstall old versions of the |bi| (optional).
```
Comment (i80and) [inline source/includes/steps-install-bi-connector_fixed.yaml]: Erm, what is this file?

**Comment by atsansone** [comment]
@i80and : Back to you. If you approve, I'll forward to @jyemin.

**Comment by atsansone** [comment]
@craiggwilson : Can you look at this change? It just adds and updates the Windows install instructions.

**Pull request merged**